src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -31,91 +89,1189 @@ const ActorInit En_Tru_InitVars = {
     (ActorFunc)EnTru_Draw,
 };
 
-// static ColliderSphereInit sSphereInit = {
-static ColliderSphereInit D_80A8B2A0 = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_SPHERE, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+#include "overlays/ovl_En_Tru/ovl_En_Tru.c"
+
+static Vec3f D_80A8B250 = { 0.0f, 0.02f, 0.0f };
+
+static Color_RGBA8 D_80A8B25C[] = {
+    { 255, 20, 60, 0 },   { 255, 235, 160, 0 }, { 20, 60, 255, 0 },
+    { 100, 100, 100, 0 }, { 255, 255, 200, 0 }, { 100, 100, 100, 0 },
+};
+
+static f32 D_80A8B274[] = { 60.0f, 255.0f, 60.0f };
+
+static UNK_TYPE D_80A8B280[] = {
+    0x0408F7E0, 0x0408F3E0, 0x0408EFE0, 0x0408EBE0, 0x0408E7E0, 0x0408E3E0, 0x0408DFE0, 0x0408DBE0,
+};
```

Comment:
These are symbols from `gameplay_keep`, I would guess textures from the way they're segmented-to-virtualled, but this needs to be checked, and proper symbols used.